ichnaea/data/tests/test_validation.py

```diff
@@ -256,7 +231,161 @@ def test_asu_signal_field_mix_up(self):
         (measure, cell) = self.make_cell_submission(asu=-75, signal=0)
         self.check_normalized_cell(measure, cell, {'signal': -75})
 
-    def test_normalize_wifis(self):
+    def test_cid_65535_without_a_valid_lac_sets_cid_to_invalid(self):
+        (measure, cell) = self.make_cell_submission(cid=65535, lac=-1, psc=1)
+        self.check_normalized_cell(measure, cell, {'cid': -1})
+
+    def test_CDMA_cell_records_must_have_MNC_MCC_LAC_CID(self):
```

Comment:
uppercase letters look weird to me here, I think we lowercased cdma, mcc, etc in test names. Also maybe use `test_cdma_records_must_have_complete_id`, or at least sort mcc, mnc in their usual order :)